src/v3/jest.config.js

```diff
@@ -22,7 +22,7 @@ const esModules = [
   '@okta/odyssey-react-mui',
   '@adobe/leonardo-contrast-colors',
   'apca-w3',
-  'colorparsley'
+  'colorparsley',
```

Comment:
Unrelated lint fix that I pulled into this PR, can disregard for review purposes